src/deviceAPI/process/functionality/image/createFromURI.js

```diff
@@ -20,7 +20,7 @@ const AuthorizedDeviceAPI = require('../../../AuthorizedDeviceAPI')
 
 module.exports = serializeAxiosResponse(
   (processId, functionalityId, body) => AuthorizedDeviceAPI.post(
-    `/processes/${processId}/functionalities/${functionalityId}/images`,
-    { body, ...DEFAULT_REQ_OPTS },
+    `processes/${processId}/functionalities/${functionalityId}/images`,
+    { ...body }, { ...DEFAULT_REQ_OPTS },
```

Comment:
This syntax seems wierd: 

`{ ...body }` is the same as jsut passing `body` (expect that you are creating a shallow clone, but I don't think this is the intent here)

Reply:
Good catch! I tink the opts part was meant for passing in additional opts which we dont use so I just changed it for it now, also taken out the destructuring from the body and tested it and it all seems to be going through 

Thanks!